src/app/visual/date-range-picker/date-range-picker-visual.component.html

```diff
@@ -0,0 +1,104 @@
+<div
+  class="date-range-demo"
+>
+  <form
+    id="screenshot-date-range-picker"
+    novalidate
+    [formGroup]="reactiveForm"
+  >
+    <div class="sky-form-group">
+      <sky-date-range-picker
+        formControlName="lastDonation"
+        label="Last donation"
+        [calculatorIds]="calculatorIds"
+        [dateFormat]="dateFormat"
+      >
+      </sky-date-range-picker>
```

Comment:
I'm seeing some strange behavior with the visual demo. Whenever I switch to "before", two fields are showing. Here's how to replicate:

1. click “set specific range”
2. change dropdown to "Before"
3. only one input *should* show. However, both are shown (you might have to do this twice to see the issue).

Reply:
Great catch. Done.